ocs_ci/ocs/ui/deployment_ui.py

```
@@ -439,8 +442,18 @@ def install_lso_cluster(self):
     def wait_next_button_lso(self):
+        next_loc = self.dep_loc.get("next_button_element", self.dep_loc["next"])
         try:
-            self.do_click(self.dep_loc["next"], enable_screenshot=True, timeout=20)
+            self.do_click(next_loc, enable_screenshot=True, copy_dom=True, timeout=20)
+        except ElementClickInterceptedException:
+            logger.warning(
+                "Next button click intercepted, retrying with JavaScript click"
+            )
+            try:
+                self.click_with_script(next_loc)
+            except Exception as e:
```

Ruff (0.16.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)